data/crac/crac-util/src/test/java/com/powsybl/openrao/data/cracutil/CracAliasesCreatorTest.java

```diff
@@ -25,35 +28,14 @@ class CracAliasesCreatorTest {
 
     private final Network network = Network.read("case-for-aliases.uct", getClass().getResourceAsStream("/case-for-aliases.uct"));
 
-    @Test
-    void testDeprecatedCracExtensions1() {
+    @ParameterizedTest
+    @ValueSource(strings = {"deprecated-crac-for-aliases-1.json", "deprecated-crac-for-aliases-2.json", "deprecated-crac-for-aliases-3.json", "deprecated-crac-for-aliases-4.json"})
+    void testDeprecatedCracExtensions(String fileName) throws IOException {
         // Extensions have been deprecated
-        InputStream inputStream = getClass().getResourceAsStream("/deprecated-crac-for-aliases-1.json");
-        OpenRaoException exception = assertThrows(OpenRaoException.class, () -> Crac.read(inputStream, network));
-        assertEquals("Extensions are deprecated since CRAC version 1.7", exception.getMessage());
-    }
-
-    @Test
-    void testDeprecatedCracExtensions2() {
-        // Extensions have been deprecated
-        InputStream inputStream = getClass().getResourceAsStream("/deprecated-crac-for-aliases-2.json");
-        OpenRaoException exception = assertThrows(OpenRaoException.class, () -> Crac.read(inputStream, network));
-        assertEquals("Extensions are deprecated since CRAC version 1.7", exception.getMessage());
-    }
-
-    @Test
-    void testDeprecatedCracExtensions3() {
-        // Extensions have been deprecated
-        InputStream inputStream = getClass().getResourceAsStream("/deprecated-crac-for-aliases-3.json");
-        OpenRaoException exception = assertThrows(OpenRaoException.class, () -> Crac.read(inputStream, network));
-        assertEquals("Extensions are deprecated since CRAC version 1.7", exception.getMessage());
-    }
-
-    @Test
-    void testDeprecatedCracExtensions4() {
-        // Extensions have been deprecated
-        InputStream inputStream = getClass().getResourceAsStream("/deprecated-crac-for-aliases-4.json");
-        OpenRaoException exception = assertThrows(OpenRaoException.class, () -> Crac.read(inputStream, network));
-        assertEquals("Extensions are deprecated since CRAC version 1.7", exception.getMessage());
+        InputStream inputStream = getClass().getResourceAsStream("/" + fileName);
+        CracCreationContext cracCreationContext = Crac.readWithContext(fileName, inputStream, network);
+        assertFalse(cracCreationContext.isCreationSuccessful());
+        assertEquals(List.of("[ERROR] Extensions are deprecated since CRAC version 1.7"), cracCreationContext.getCreationReport().getReport());
+        // TODO : instead of failing import, now that we have CracCreationContext with Json CRAC, ignore extensions and log the issue
```

Comment:
no because we still throw an exception and stop importing the json. the idea would be to log the issue in the context (currently not used by json deserializers) and continue on importing